src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -657,23 +659,37 @@ private void processTile(
         return;
       }
 
-      int bytesPerPixel = FormatTools.getBytesPerPixel(pixelType);
-      switch (bytesPerPixel) {
-        case 1:
+      ByteBuffer bb = ByteBuffer.wrap(tile);
+      if (resolution == 0 && isLittleEndian) {
+        bb = bb.order(ByteOrder.LITTLE_ENDIAN);
+      }
+      switch (pixelType) {
+        case FormatTools.INT8:
+        case FormatTools.UINT8: {
           dataBlock = new ByteArrayDataBlock(size, gridPosition, tile);
           break;
-        case 2:
+        }
+        case FormatTools.INT16: {
```

Comment:
Add ```FormatTools.UINT16``` too?

Reply:
Ha! Was just about to say I had run into this: ```loci.formats.FormatException: Unsupported pixel type: uint16```. Thanks, guys.